chef/data_bags/crowbar/template-crowbar.json

```diff
@@ -28,10 +28,10 @@
     "crowbar": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 11,
+      "schema-revision": 100,
       "element_states": {
         "crowbar": [ "all" ],
-        "crowbar-upgrade": [ "crowbar_upgrade" ]
+        "crowbar-upgrade": [ "readying", "ready", "applying", "crowbar_upgrade" ]
```

Comment:
Probably yes, but I was more confident when I just used the same value as in Cloud5